Source/WebKit/WebProcess/GPU/graphics/WebGPU/RemoteTextureProxy.cpp

```diff
@@ -70,14 +70,18 @@ Ref<WebCore::WebGPU::TextureView> RemoteTextureProxy::createView(const std::opti
 
 void RemoteTextureProxy::destroy()
 {
-    auto sendResult = send(Messages::RemoteTexture::Destroy());
-    UNUSED_VARIABLE(sendResult);
+    if (m_createdView) {
+        auto sendResult = send(Messages::RemoteTexture::Destroy());
+        UNUSED_VARIABLE(sendResult);
+    }
 }
 
 void RemoteTextureProxy::setLabelInternal(const String& label)
 {
-    auto sendResult = send(Messages::RemoteTexture::SetLabel(label));
-    UNUSED_VARIABLE(sendResult);
+    if (m_createdView) {
```

Comment:
I don't think this is correct because we should be able to set the label on a GPUTexture before creating its view